packages/nekoton_bridge/native/src/utils/caller.rs

```diff
@@ -16,16 +17,21 @@ use uuid::Uuid;
 
 use crate::utils::mega_struct;
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Error)]
 pub enum ErrorCode {
+    #[error("No error, ok")]
     Ok,
+    #[error("Network error")]
     Network,
+    #[error("Some error in logic")]
     Generic,
 }
 
 // / Dynamic value for transmitting between Dart and Rust. We can't use Box<dyn Any> because frb doesn't support it.
 #[derive(Clone, Debug)]
 pub enum DynamicValue {
+    U16(u16),
```

Comment:
maybe we should add
```dart
if (this is DynamicValue_U16) return (this as DynamicValue_U16).field0;
```
to dynamic_value.dart?

Reply:
-